Chapter09/src/main/kotlin/2_Apply.kt

```diff
@@ -0,0 +1,14 @@
+fun main() {
+    val `007` = JamesBond().apply {
+        name = "Sean Connery"
+        movie = "Dr. No"
+    }
+
+    println(`007`.name)
+}
+
+class JamesBond {
+    lateinit var name: String
+    lateinit var movie: String
+    lateinit var alsoStarring: String
```

Comment:
This leads to UninitializedPropertyAccessException if property alsoStarring is accessed. For having correct code I would also initialize this property.